configuration/csv-to-docs.md

```diff
@@ -1,32 +1,276 @@
+  "parent": {
+    "type": "district_hospital",
+    "parent": "",
+    "is_name_generated": "false",
+    "name": "D2",
+    "external_id": "",
+    "notes": "",
+    "geolocation": "",
+    "reported_date": 1544031155715,
+    "_id": "f223f240-5d6a-5a7a-91d4-46d3c59de73e"
+  },
+  "is_name_generated": "false",
+  "name": "HC7",
+  "external_id": "",
+  "notes": "",
+  "geolocation": "",
+  "reported_date": 1544031155715,
+  "_id": "480d0cd0-c021-5d55-8c63-d86576d592fc"
 }
 ```
 
-### Commands to Fetch and Upload Docs
+#### Get Value From Another Doc
 
-Run command `medic-conf fetch-csvs-from-google-drive`. This will download the CSV files in the json document created above and place them into a folder named `csv`.
+Similar to including another doc, it is also possible to get the value of a specific field in another doc. For instance, if `parent:GET _id OF place WHERE reference_id=COL_VAL` were used in the example above, the `parent` field's value would have been set to the `_id` of the refered to doc instead of including the whole doc. Note that `_id` is a generated value included in all generated docs.
 
-Running `medic-conf csv-to-docs` will convert the CSV into json docs for the webapp. They are placed in a `json_docs` folder
+| reference_id:excluded | parent:GET _id OF place WHERE reference_id=COL_VAL | is_name_generated | name | reported_date:timestamp |
+| --------------------- | -------------------------------------------------- | ----------------- | ---- | ----------------------- |
+| health_center_1       | district_1                                         | false             | HC1  | 1544031155715           |
+| health_center_2       | district_2                                         | false             | HC2  | 1544031155715           |
+| health_center_3       | district_3                                         | false             | HC3  | 1544031155715           |
+
+The resulting doc structure would be:
+
+```
+{
+  "type": "health_center",
+  //Parent property with the _id from district_1 as the value.
+  "parent": "0c31056a-3a80-54dd-b136-46145d451a66",
+  "is_name_generated": "false",
+  "name": "HC3",
+  "external_id": "",
+  "notes": "",
+  "geolocation": "",
+  "reported_date": 1544031155715,
+  "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
+}
+```
 
-Lastly run `medic-conf --local upload-docs` to upload the converted docs into your local instance. `--local` can be replaced with an instance or URL. See [medic-conf](https://github.com/medic/medic-conf) for detailed instructions.
+## Creating CSV files for Users
 
-### Google Drive Authentication
 
-Medic-conf leverages google authentication to access google drive. You will need to create a client_secrets file named `.gdrive.secrets.json` and place it in the same directory where you want the CSVs to be fetched. [Token Creation](https://developers.google.com/identity/protocols/OAuth2InstalledApp)
+### Creating a new user with a new place
```

Comment:
In this heading we're talking about a new place, and in the next heading we talk about linking users to contacts. As these are supposed to be two different variants of achieving the same thing we should keep the naming consistent, for example:
 - Creating a new user with a new place
 - Creating a new user with an existing place

Reply:
Good call! Just to clarify here, the content isn't about an existing place, but rather a person or place defined in a CSV.

```suggestion
### Creating a new user linked to contacts in CSV files
```